libnmstate/nm/applier.py

```diff
@@ -175,11 +175,7 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
                         device.deactivate,
                         device.delete,
                     ]
-                    if nmdev.get_device_type() in (
-                        nmclient.NM.DeviceType.OVS_BRIDGE,
-                        nmclient.NM.DeviceType.OVS_PORT,
-                        nmclient.NM.DeviceType.OVS_INTERFACE,
-                    ):
+                    if nmdev.is_software():
```

Comment:
This seems to be true for Veth-Interfaces as well but removing veths is not appropriate in my opinion.

Reply:
It is appropriate, but it does not work well with the test env.
We should support veth interfaces and with it the ability to remove them.

Reply:
@almusil suggested to do this only for `ABSENT` and not `DOWN`.

Reply:
Done